ports/esp32/partitions-4MiB.csv

```diff
@@ -0,0 +1,7 @@
+# Notes: the offset of the partition table itself is set in
```

Comment:
We should rename the existing `partitions.csv` to `partitions-4MiB.csv`.

Reply:
@mattytrentini are you still interested in adding this board?  If so this PR needs rebasing and this file needs to be removed (it now exists in the repo).